src/Umbraco.Tests/Services/EntityXmlSerializerTests.cs

```diff
@@ -56,10 +56,10 @@ public void Can_Export_DictionaryItems()
         public void Can_Export_Languages()
         {
             // Arrange
-            var languageNbNo = new Language("nb-NO") { CultureName = "Norwegian" };
+            var languageNbNo = new Language("nb-NO", "Norwegian");
```

Comment:
This culture name needs to be exactly `Norwegian`, so it matches the XML used in the assertion of this test (or we could update the XML, but that's probably also used in other tests).

Reply:
Looks like none of the other tests use the culture name stored in the XML. I've updated it and pushed a new commit to re-run the tests...